app-drawer-layout/app-drawer-layout.html

```diff
@@ -224,17 +139,8 @@
       },
 
       attached: function() {
-        this.fire('iron-resize');
-        this.shouldEnableScroll(true, this.$.drawerContainer);
-      },
-
-      /**
-       * Toggles the drawer panel open and closed.
-       *
-       * @method toggleDrawer
-       */
-      toggleDrawer: function() {
-        this.opened = !this.opened;
+        this.listen(this.drawer, 'app-drawer-position-changed', '_resetLayout');
```

Comment:
this assumes the drawer is there at `app-layout-drawer`'s attached time.  It is possible that the drawer is appended dynamically and this will not work.  How about just add the `app-drawer-position-changed` to the `listeners` object?